lib/pages/common_widgets/goal_widget.dart

```diff
@@ -24,6 +27,24 @@ class GoalWidget extends StatelessWidget {
       });
     }
 
+    doFinishGoal() {
+      final Future<Map<String, dynamic>> responseMessage = finishOrExtend(context, goal.goalId, "true", "0");
+      responseMessage.then((response) {
+        if (response['status'] == true) {
+        onFinish();
+        } else {}
+      });
+    }
+
+     doExtendGoal() {
+      final Future<Map<String, dynamic>> responseMessage = finishOrExtend(context, goal.goalId, "false", (goalExtensionInput.text.toString()));
```

Comment:
you can just have false without "", since its a bool

Reply:
Assuming that's how it is on the back end as well

Reply:
see comment about goalStatus in other conversation, same issue